beanie/odm/documents.py

```diff
@@ -210,7 +259,7 @@ def find_many(
         :param session: Optional[ClientSession] - pymongo session
         :return: [FindMany](https://roman-right.github.io/beanie/api/queries/#findmany) - query instance
         """
-        return FindMany(document_model=cls).find_many(
+        return FindMany[DocType](document_model=cls).find_many(  # type: ignore
```

Comment:
@roman-right 

That is one of the two lines I had to disable.

That's because mype throws error like "too few agruments (2 expected)" but that is wrong, it expect just one.

I tried to pass 2 equals arguments, but it fails at runtime